<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logwidget</class>
 <widget class="QWidget" name="Logwidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="logonlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>861</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>761</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="namelineEdit">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordlineEdit">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="namelabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;User:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordlabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="namepushButton">
        <property name="text">
         <string>keyboard</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="passwordpushButton">
        <property name="text">
         <string>keyboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>100</number>
      </property>
      <property name="leftMargin">
       <number>200</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item>
       <widget class="QCheckBox" name="namecheckBox">
        <property name="text">
         <string>RememberU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="passwordcheckBox">
        <property name="text">
         <string>RememberP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>100</number>
      </property>
      <property name="leftMargin">
       <number>150</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>150</number>
      </property>
      <item>
       <widget class="QPushButton" name="longonpushButton">
        <property name="text">
         <string>login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetpushButton">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
